tests/test_daemon_script.py

```diff
@@ -214,7 +215,14 @@ def setUp(self):
         self.conf_patcher = mock.patch('geospaas_harvesting.harvest.Configuration')
         self.conf_mock = self.conf_patcher.start()
 
+        self.patcher_param_count = mock.patch.object(Parameter.objects, 'count')
+        self.mock_param_count = self.patcher_param_count.start()
+        self.mock_param_count.return_value = 2
```

Comment:
Since you added the update of vocabularies in `main()` in `harvest.py`:
- the `MainTestCase` class must now inherit from `django.test.TestCase` and not `unittest.TestCase`. Otherwise it can run on it's own because the vocabularies tables are not initialized. Right now the tests pass when all tests are run because the database is initialized by other test classes running before this one.
- The actual vocabularies update should not run during the tests. It's not what we are  testing, and it's very time-consuming, so it should be mocked.

Reply:
Done in the next commit